util.cpp

```diff
@@ -79,6 +79,29 @@ namespace Sass {
     return out;
   }
 
+  // read css string (handle multiline DELIM)
+  string read_css_string(const string& str)
+  {
+    string out("");
+    bool esc = false;
+    for (auto i : str) {
+      if (i == '\\') {
+        esc = ! esc;
+      } else if (esc && i == '\r') {
```

Comment:
This is failing sass/sass-spec#278 on Windows because of `\r\n`. If I convert the endings using `dos2unix test/fixtures/spec/spec/libsass-closed-issues/issue_942/input.scss` it passes. But I suppose we don't users to use the `dos2unix` utility before compilation. :)